docker-for-windows/troubleshoot.md

```diff
@@ -122,36 +122,21 @@ deployment
 requirements?](faqs.md#can-i-change-permissions-on-shared-volumes-for-container-specific-deployment-requirements)
 in the FAQs.
 
-#### inotify on shared drives does not work
-
-Currently, `inotify` does not work on Docker Desktop. This becomes evident,
-for example, when an application needs to read/write to a container across a
-mounted drive. Instead of relying on filesystem inotify, we recommend using
-polling features for your framework or programming language.
-
-* **Workaround for nodemon and Node.js** - If you are using
-  [nodemon](https://github.com/remy/nodemon) with `Node.js`, try the fallback
-  polling mode described here: [nodemon isn't restarting node
-  applications](https://github.com/remy/nodemon#application-isnt-restarting).
-
-* **Docker Desktop for Windows issue on GitHub** - See the issue [Inotify on shared drives does not work](https://github.com/docker/for-win/issues/56#issuecomment-242135705).
-
-#### Volume mounting requires shared drives for Linux containers
+#### Volume mounting requires shared folders for Linux containers
 
 If you are using mounted volumes and get runtime errors indicating an
 application file is not found, access is denied to a volume mount, or a service
 cannot start, such as when using [Docker Compose](/compose/gettingstarted.md),
-you might need to enable [shared drives](index.md#shared-drives).
+you might need to enable [shared folders](index.md#file-sharing).
 
-Volume mounting requires shared drives for Linux containers (not for Windows
+Volume mounting requires shared folders for Linux containers (not for Windows
 containers). Click ![whale menu](images/whale-x.png){: .inline}
- and then **Settings** > **Shared Drives** and share the drive that contains the
+ and then **Settings** > **Shared Folders** and share the folder that contains the
 Dockerfile and volume.
 
 #### Understand symlinks limitations
 
-Symlinks work within and across containers. However, symlinks created outside of
-containers (for example, on the host) do not work. To learn more, see [Are
+Symlinks work within and across containers. To learn more, see [Are
```

Comment:
So, they are no limitations now, are they? Maybe we should change the section title?